web_console/pages/job/index.jsx

```diff
@@ -76,12 +76,12 @@ function useStyles(theme) {
   `;
 }
 
-function JobList(props) {
+export default function JobList(props) {
   const theme = useTheme();
   const styles = useStyles(theme);
 
   const { data, mutate } = useSWR('jobs', fetcher);
-  const jobs = data ? data.data.filter(x => x.metadata) : null;
+  const jobs = data ? data.data : null
```

Comment:
keep the filter, or the page would crash. the whole page is heavily rely on k8s metadata currently

Reply:
a mistake here